eventloghook.go

```diff
@@ -7,16 +7,16 @@ import (
 	"os"
 
 	"github.com/sirupsen/logrus"
-	"golang.org/x/sys/windows/svc/debug"
+	"golang.org/x/sys/windows/svc/eventlog"
 )
 
 // EventLogHook to send logs via windows log.
 type EventLogHook struct {
-	upstream debug.Log
+	upstream eventlog.Log
 }
 
-// NewHook creates and returns a new EventLogHook wrapped around anything that implements the debug.Log interface
-func NewHook(logger debug.Log) *EventLogHook {
+// NewHook creates and returns a new EventLogHook wrapped around anything that implements the eventlog.Log interface
+func NewHook(logger eventlog.Log) *EventLogHook {
```

Comment:
This is a breaking change really, debug.Log was an interface so it accepted the pointer from `event log.Open()` but `eventlog.Log` is a type and so this would need to accept a pointer.

It may work with passing in a copy of the the `eventlog.Log` but so probably the readme should be updated if thats the case.

I think accepting a pointer here is better

Reply:
https://github.com/freman/eventloghook/pull/5

Reply:
Hi there,

First of all, I'm so sorry I didn't even think to check the backwards compatibility... I was still sure that was an interface.
Second, I'd like to propose a compromise: https://github.com/freman/eventloghook/pull/6